README.md

```diff
@@ -14,6 +14,17 @@
 
 JSON Schemas are the SSoT (Single Source of Truth) that allows you to share form's _structure_ and _validations_ between frontend and backend, regardless of the language used.
 
+## Installation
+
+Available on [NPM](https://www.npmjs.com/package/@remoteoss/json-schema-form).
```

Comment:
note: You don't imagine how many times I had to google npm json-schema-form today. I thought adding the link here would make things much easier. WDYT?

Reply:
Makes sense to me :)